G/GoogleSparseHash/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GoogleSparseHash"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Andersen98/sparsehash-c11.git", "24e94b66249466f71f67a54a96247d99b8dcceb8")
+]
+
+
+script = raw"""
+cd sparsehash-*
+install_license LICENSE
+
+mkdir ${prefix}/include
+mv src/sparsehash ${prefix}/include/
```

Comment:
```suggestion
cd sparsehash-*
install_license LICENSE

mkdir ${prefix}/include
mv sparsehash ${prefix}/include/
```

This is just a header only library so I think all I need to do is just copy over the files rather than build anything. Let me know if this isn't the case!